index.html

```diff
@@ -450,14 +450,14 @@
 						<td>Sync</td>
 						<td>Yes</td>
 						<td>Yes</td>
-						<td>Varies:  when the pointer is primary, all default actions of <code>mouseover</code></td>
+						<td>None</td>
 					</tr>
```

Comment:
Please call out these changes in the description also (since they're only tangentially related).  I.e. that since UI Events says these mouse events have no default action, we can just say the same for the pointer events.